dictionaries/companies/src/companies.txt

```diff
@@ -774,6 +776,8 @@ Illinois Tool Works Inc.
 Illumina (company)
 Illumina Inc
 ILMN
+imessage
```

Comment:
Is `imessage` a company?

Reply:
I think it is better to move it to [cspell-dicts/software-services.txt](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/software-services.txt)

Reply:
Good point.
I added it in https://github.com/streetsidesoftware/cspell-dicts/pull/1941